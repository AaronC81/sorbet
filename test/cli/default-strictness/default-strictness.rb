"x" + :error

module Foo
  extend T::Sig

  sig { returns(Integer) }
  def bar
    3
  end
end
