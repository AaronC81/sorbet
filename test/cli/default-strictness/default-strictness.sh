#!/bin/sh
main/sorbet --silence-dev-message --default-strictness true test/cli/default-strictness/default-strictness.rb 2>&1