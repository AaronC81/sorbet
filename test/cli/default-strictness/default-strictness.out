test/cli/default-strictness/default-strictness.rb:1: Expected `String` but found `Symbol(:"error")` for argument `arg0` https://srb.help/7002
     1 |"x" + :error
        ^^^^^^^^^^^^
    https://github.com/sorbet/sorbet/tree/master/rbi/core/string.rbi#L58: Method `String#+` has specified `arg0` as `String`
    58 |        arg0: String,
                ^^^^
  Got Symbol(:"error") originating from:
    test/cli/default-strictness/default-strictness.rb:1:
     1 |"x" + :error
              ^^^^^^
Errors: 1
